ironfish/src/defaultNetworkDefinitions.ts

```diff
@@ -70,104 +70,50 @@ export const MAINNET = `
 }
  `
 
-export const TESTNET_PHASE_2 = `
- {
-    "id": 2,
-    "bootstrapNodes": [
-        "test.bn1.ironfish.network"
-    ],
-    "genesis": {
-      "header": {
-        "sequence": 1,
-        "previousBlockHash": "0000000000000000000000000000000000000000000000000000000000000000",
-        "noteCommitment": {
-          "type": "Buffer",
-          "data": "base64:wQugqoAkjjargzkuEqn+LSXxAvlMp5ubeAgHqpUmrAw="
-        },
-        "transactionCommitment": {
-          "type": "Buffer",
-          "data": "base64:KYJk06s6m1IblZiINE3U6cGN8YLwNpmawcr7N1XoqCE="
-        },
-        "target": "883423532389192164791648750371459257913741948437809479060803100646309888",
-        "randomness": "0",
-        "timestamp": 1669940355519,
-        "graffiti": "67656E6573697300000000000000000000000000000000000000000000000000",
-        "noteSize": 3,
-        "work": "0",
-        "nullifierSize": 1
+export const DEV = `{
```

Comment:
Updating the `DEV` to be the same as `TESTING` because I think there may have been some block changes that went in after this was last updated